src/applications/personalization/profile/components/alerts/bad-address/ProfileAlert.jsx

```diff
@@ -21,7 +22,7 @@ const handlers = {
   },
 };
 
-export default function ProfileAlert() {
+export default function ProfileAlert({ className = 'vads-u-margin-top--4' }) {
```

Comment:
Non-blocking q, out of curiosity: is there a benefit to using this function declaration instead of the ES6 arrow way?

Reply:
no I dont think there is a benefit. I usually use arrow functions out of habit, and bc I don't like having to deal with function hoisting from regular functions. This is probably just a regular function because I didn't build it originally.